src/components/Pagination.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, Center } from "@chakra-ui/react";
+import React from "react";
+
+interface PaginationProps {
+  paginationPages: number;
+  activePage: number;
+  handlePaginationPageChange: (paginationPages: number) => void;
+}
+
+export const Pagination = ({
+  handlePaginationPageChange,
+  paginationPages,
+  activePage,
+}: PaginationProps) => {
+  const buttonList: JSX.Element[] = [];
+  for (let i = 0; i < paginationPages; i++) {
```

Comment:
You can start loop from 1, so you don't need to have `if (paginationPages !== 0)`